python/triton/runtime/jit.py

```diff
@@ -36,6 +36,9 @@ def ret(self):
         return self.hasher.hexdigest()
```

Comment:
@Sarbojit2019 Please correct your reproducer so that we can use it without any changes

Reply:
@Jokeren, could you please help me understand the issue you see with the reproducer?

Reply:
We cannot run your reproducer directly. There are at least two problems IIRC. For example, `e_src` is not defined.

Reply:
Take a quick look, another one should be this:

```
    num_rows, num_indices, heads = 16, 16, 16, 4
```